cmd/grpc-app/main.go

```diff
@@ -1,18 +1,36 @@
 package main
 
 import (
-	"fmt"
+	"log"
 
+	cmd "github.com/go-seidon/local/cmd"
 	grpc_app "github.com/go-seidon/local/internal/grpc-app"
+	repository_mongo "github.com/go-seidon/local/internal/repository-mongo"
 )
 
 func main() {
-	app, err := grpc_app.NewGrpcApp()
+	config := cmd.Config{
+		GrpcPort:            8005,
+		MongoHost:           "localhost",
+		MongoPort:           27017,
+		MongoConnectTimeout: 1000,
+		FileDirectory:       "/Users/yudi_s/delete-me",
+	}
+
+	mongoClient, ctx := repository_mongo.NewMongoClient(config)
+	fileRepo := repository_mongo.NewFile(mongoClient)
```

Comment:
I have no strong opinion but the reason I decide to choose `NewFile` is because the package name `repository_mongo` has already told the context.

But we agree that the best naming is the shortest naming that people would understand. Do you think NewFile is hard to grasp? If so, I'd happy to change the name to `NewFileRepository`.

Reply:
After this conversation, I encourage us to document the naming convention